core/bind/core_bind.cpp

```diff
@@ -1251,78 +1251,138 @@ bool _File::is_open() const {
 }
 
 String _File::get_path() const {
-	ERR_FAIL_COND_V_MSG(!f, "", "File must be opened before use.");
+#ifdef ANDROID_ENABLED
+	ERR_FAIL_COND_V_MSG(!f, String(), "App should have read-write permission and file must be opened before use.");
```

Comment:
Small optimization hint: can't it be extracted as a `const String` at the top of the file to avoid so many String duplicates?

Reply:
Can be done but I wasn't sure if I should do it. A  dev reading it would need to scroll to beginning of the file to look for it. I'll make the changes if that's fine.

Reply:
We can call the constant string `android_hint` I guess.